ansible/roles.yml

```diff
@@ -0,0 +1,4 @@
+- src: City-of-Bloomington.linux
```

Comment:
are these files missing from the commit? Also, naming seems a bit customer/branch specific? Did I miss something, I have no experience of ansible?

Reply:
The files are all there in the commit.  They are all in an "ansible" directory at the root of the project.  The idea is to be able to clone the project, then use ansible to deploy it to a server.  Admittedly, this ansible work is targetting an Ubuntu 16.04 server, but it's a start.

Reply:
This is the ansible work we did to deploy ASM on our servers back in June.  The commit is a little out of date now, as the ansible roles it depends on have changed.

For our web applications, we have found it useful to have deployment scripts included as part of the project. (As opposed to a seperate Github repository).  Is this a strategy that sounds like it would be useful for you?

Reply:
there are 10 files in this commit and none of them are called City-of-Bloomington.anything ? It looks like this roles.yml file is referencing additional files not in the commit (or are they role names rather than files and I've misunderstood?). 

I'm fine including deployment-related stuff in the repo, Ansible looks very interesting and I'm sure there are similar automation products like puppet and chef that are popular too.  I just wanted to make sure that everything was there before merging and that the names were ok.

If we're good I'll merge it in so that further mods can be made to it. Apologies for taking so long to spot this pull request :)

Reply:
No worries, thanks for reviewing it.  You are correct the role dependencies are not included (all the City-of-Bloomington.*) directories.  They are pulled down using ansible's dependency management system, Galaxy.

```bash
ansible-galaxy install -r roles.yml
ansible-playbook asm_install.yml -i /path/to/your/inventory
```